camp/asher-miniproject/src/main/java/frc/robot/subsystems/MotorSubsystem.java

```diff
@@ -0,0 +1,147 @@
+        return wrapDegrees(motorEncoder.getPosition().getValueAsDouble()*360);
+    }
+
+    /**
+     * Clamps a given value within the specified range [min, max].
+     * If the value is less than the minimum, the minimum is returned.
+     * If the value is greater than the maximum, the maximum is returned.
+     * Otherwise, the value itself is returned.
+     *
+     * @param value The value to be clamped.
+     * @param min   The minimum allowable value.
+     * @param max   The maximum allowable value.
+     * @return The clamped value within the range [min, max].
+     * @throws IllegalArgumentException if the minimum value is greater than the maximum value.
+     */
+    public static double clamp(double value, double min, double max) {
+        if (min > max) {
+            throw new IllegalArgumentException("min must not be greater than max");
+        }
+
+        return max(min, min(value, max));
+    }
+
+    private void configureAngleEncoder() {
+        // Config CanCoder
+        CANcoderConfiguration encoderConfigs = new CANcoderConfiguration();
+        encoderConfigs.MagnetSensor.MagnetOffset = degreesToRotations(Constants.encoderAbsoluteOffsetDegrees);
+        encoderConfigs.MagnetSensor.AbsoluteSensorDiscontinuityPoint = Constants.absoluteSensorDiscontinuityPoint;
+        encoderConfigs.MagnetSensor.SensorDirection = Constants.shooterAngleSensorDirection;
+        // apply configs
+        StatusCode response = motorEncoder.getConfigurator().apply(encoderConfigs);
+        if (!response.isOK()) {
+        DataLogManager.log(
+            "CANcoder ID " + motorEncoder.getDeviceID() + " failed config with error " + response.toString());
+        }
+    }
+
+    private void configureMotor() {
+        TalonFXConfiguration config = new TalonFXConfiguration();
+        config.MotorOutput.NeutralMode = NeutralModeValue.Brake;
+
+        config.Voltage.PeakForwardVoltage = Constants.falconMaxVoltage;
+        config.Voltage.PeakReverseVoltage = -Constants.falconMaxVoltage;
+        config.Voltage.SupplyVoltageTimeConstant = Constants.motorSupplyVoltageTimeConstant;
+
+        config.CurrentLimits.StatorCurrentLimit = Constants.motorStatorCurrentMaximumAmps;
+        config.CurrentLimits.StatorCurrentLimitEnable = true;
+        config.CurrentLimits.SupplyCurrentLimit = Constants.motorSupplyCurrentMaximumAmps;
+        config.CurrentLimits.SupplyCurrentLimitEnable = true;
+
+        config.MotorOutput.Inverted = InvertedValue.Clockwise_Positive;
+
+        StatusCode response = motor.getConfigurator().apply(config);
+        if (!response.isOK()) {
+            System.out.println(
+                    "TalonFX ID " + motor.getDeviceID() + " failed config with error " + response.toString());
+        }
+    }
+
+    private double degreesToRotations(double degrees)
+    {
+      return degrees/360;
+    }
+
+    private double wrapDegrees(double degrees){
+        return (degrees % 360 + 540) % 360 - 180;
```

Comment:
please leave a comment explaining the behavior of this code. Where is the wrap point? 
Also seems like the double modulo 360 is redundant. Maybe: (degrees + 540) % 360 - 180